remappings.txt

```diff
@@ -1,3 +1,3 @@
 ds-test/=lib/forge-std/lib/ds-test/src/
 forge-std/=lib/forge-std/src/
-openzeppelin-contracts/=lib/openzeppelin-contracts/
```

Comment:
also what happened here?

Reply:
I realized it wasn't needed when I was cleaning up the remappings.
Everything is imported with `"openzeppelin-contracts/..."` and Foundry automatically reads from the lib folder.